packages/core/src/components/canAccess/index.tsx

```diff
@@ -25,6 +25,19 @@ type CanAccessBaseProps = {
      * Content to show if access control returns `false`
      */
     fallback?: React.ReactNode;
+    /**
+     * Callback function to be called if access control returns `false`
+     */
+    onUnauthorized?: (props: {
```

Comment:
Can we make `props` an interface, instead of writing them inline?

```suggestion
    onUnauthorized?: (props: OnUnauthorizedProps)
```

Reply:
🚀